### PR TITLE
Handle Keyword Parameters

### PR DESCRIPTION
**Pull request by scbedd**
[Test Build Consuming docfx-yaml from my fork via git+https](https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=150344&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=70b92200-87db-50c8-35f9-975ba5f9e5b7)
[Output](https://github.com/MicrosoftDocs/azure-docs-sdk-python/tree/smoke-test-preview-docfx)

Waiting for publishing to complete to review it on review.docs.ms. @joelmartinez other than providing an easy diff between the original and new intermediate yml output, what other testing do you look for here?

### CODE REVIEW

**Review by joelmartinez — CHANGES_REQUESTED** [review]
Hey @scbedd ... yeah, really all we're looking to establish is

1. That nothing has regressed ... and
2. An example of the `keyword` parameter is translated correctly onto the published page

Aside from that, this looks like a pretty small/simple change ... only thing I'd ask aside from the verification of the published page once the build is done, is to update the version number of the package so it can be published to PyPI :)

**Comment by scbedd** [comment]
@joelmartinez re:version bump: 100%!

> That nothing is regressed

That's more what my question was based around. How can I tell if something is regressed? Outside of diffing a specific package from like `smoke-test-preview` to `smoke-test-preview-docfx` VISUALLY, I'm not certain how I'm going to confirm that. I'll definitely be executing on at least the above check, but do you have any better recs?

**Comment by joelmartinez** [comment]
@scbedd you hit the nail on the head that there's not really a great way of checking regressions right now ... at least not without doing some diffs of the review and live branch. But this is such a small change, that I think it will enough to just verify that existing `parameter` values are still rendered in addition to any new `keyword` values for now.

**Comment by scbedd** [comment]
I'll include direct links to before/after where keyword args are present.

**Comment by scbedd** [comment]
Hey @joelmartinez 

I did a 1:1 generation against the `preview` packages with a custom CI build.

[Example of new sphinx-docfx-yaml rendered yml](https://review.docs.microsoft.com/en-us/python/api/azure-ai-textanalytics/azure.ai.textanalytics.textanalyticsclient?view=azure-python-preview&branch=smoke-test-preview-docfx#parameters)
[Same page using CURRENT sphinx-docfx-yaml](https://review.docs.microsoft.com/en-us/python/api/azure-ai-textanalytics/azure.ai.textanalytics.textanalyticsclient?view=azure-python-preview&branch=smoke-test-preview#parameters)

Example of standard AND keyword args right there.

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='sphinx-docfx-yaml',
-    version='1.2.73',
+    version='1.2.74',
```
Comment (scbedd) [inline setup.py:16]: I consider this to be a bug that is being fixed, so I'm calling this a patch.

There are arguments to be made that this is enough of an enhancement to call this a minor version. 

Up to you @joelmartinez

Comment (joelmartinez) [reply setup.py:16]: Yeah this is fine by me :)

**Review by joelmartinez — APPROVED** [review]
This looks good ... I will probably publish this release first thing tomorrow morning. Thanks Scott!

**Comment by scbedd** [comment]
@joelmartinez thank you! I don't have write access to this repo, please merge whenever you're ready!

**Pull request merged**